project-lifecycle.md

```diff
@@ -43,40 +43,40 @@ and may go through several iterations.
 
 # Proposal
 
-Project Proposals shall be submitted to the [TSC](https://www.hyperledger.org/about/leadership)
+Project Proposals are submitted to the [TSC](https://www.hyperledger.org/about/leadership)
 for review, using [Proposal Template](https://hyperledger.github.io/hyperledger-hip/).
-Proposals that are approved shall enter into an *Incubation* state, unless
-they are of a refactoring nature, in which case they will be turned over
-to the relevant project maintainer(s) to handle as they deem fit.
+Proposals that are approved enter into an *Incubation* state, unless
+they are of a refactoring nature, in which case the project is turned over
+to the relevant project maintainers to handle as they deem fit.
 
 A Proposal must:
 
--   have a clear description
--   have a well-defined scope
--   identify committed development resources
--   identify initial maintainers
--   be vendor neutral
+-   Have a clear description
```

Comment:
suggest capitalizing the first letter in a list item, following https://developers.google.com/style/lists?hl=en#numbered-lettered-bulleted-lists